### PR TITLE
Rename entites & packages

### PR DESCRIPTION
**Pull request by adrtamas**

### CODE REVIEW

File: src/main/java/org/seatsurfer/domain/Admin.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            name = "email",
+            nullable = false,
+            columnDefinition = "TEXT"
+    )
+    private String email;
+
+    @Column(
+            name = "password",
+            nullable = false,
+            columnDefinition = "TEXT"
```
Comment (adrtamas) [inline src/main/java/org/seatsurfer/domain/Admin.java]: I am not sure if it helps to mention columnDefinition, from documentation "Defaults to the generated SQL to create a column of the inferred type."

File: src/main/java/org/seatsurfer/domain/Admin.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    private String password;
+
+    @OneToMany(
+            mappedBy = "admin",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true
+    )
+    private List<Building> buildings = new ArrayList<>();
+
+    // Add Building to Admin
```
Comment (adrtamas) [inline src/main/java/org/seatsurfer/domain/Admin.java]: We are trying to add comments only when is really really necessary
check: https://ilolo.medium.com/clean-code-101-comments-be39b5f2cfd0

File: src/main/java/org/seatsurfer/domain/Admin.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    public void addBuilding(Building building) {
+        buildings.add(building);
+        building.setAdmin(this);
+    }
+
+    // Remove Building from Admin
```
Comment (adrtamas) [inline src/main/java/org/seatsurfer/domain/Admin.java]: same

File: src/main/java/org/seatsurfer/domain/Building.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Table(
+        name = "buildings",
+        uniqueConstraints = {
```
Comment (adrtamas) [inline src/main/java/org/seatsurfer/domain/Building.java]: So this mean that I cannot have buildings with same id AND same name?

File: src/main/java/org/seatsurfer/domain/Storey.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            nullable = false,
+            columnDefinition = "TEXT"
+    )
+    private String name;
+
+    @Column(
+            name = "max_rows",
+            nullable = false
+    )
+    final private Integer maxRows = 25;
```
Comment (adrtamas) [inline src/main/java/org/seatsurfer/domain/Storey.java]: Reorder the modifiers to comply with the Java Language Specification.

File: src/main/java/org/seatsurfer/domain/Admin.java (hunk 1 above)
Comment (ProdanRaduMatei) [reply src/main/java/org/seatsurfer/domain/Admin.java]: Yes, i removed columnDefinition.

File: src/main/java/org/seatsurfer/domain/Admin.java (hunk 2 above)
Comment (ProdanRaduMatei) [reply src/main/java/org/seatsurfer/domain/Admin.java]: Yes, I agree.

**Pull request merged**